src/Tribe/Aggregator/Record/List_Table.php

```diff
@@ -742,12 +743,12 @@ protected function pagination( $which ) {
 		$current              = $this->get_pagenum();
 		$removable_query_args = wp_removable_query_args();
 
-		$current_url = set_url_scheme( 'http://' . esc_url_raw( $_SERVER['HTTP_HOST'] ) . esc_url_raw( $_SERVER['REQUEST_URI'] ) );
+		$current_url = set_url_scheme( esc_url_raw( $_SERVER['REQUEST_URI'] ), 'relative' );
```

Comment:
:no_entry_sign: Detected usage of a possibly undefined superglobal array index: $_SERVER['REQUEST_URI']. Use isset() or empty() to check the index exists before using it

<sub>hash: 5d82b03bf5bbbe4ce433506621583079</sub>